docs/receiving/receiving-with-ingest.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Receiving Webhooks with Ingest
+---
+
+import CodeTabs from '@theme/CodeTabs';
+import TabItem from '@theme/TabItem';
+
+[Svix Ingest] grants you scalability, security best practices, and
+reliability, all with minimal configuration.
+
+Creating a `Source` will generate an endpoint you can share with a webhook
+provider as a destination for their webhooks.
+
+Ingest supports a variety of signature verification schemes used by webhook
+providers. Supported values for a `Source`'s' `type` include:
+
+- `beehiiv`
+- `brex`
+- `clerk`
+- `github`
+- `guesty`
+- `hubspot`
+- `incidentIo`
+- `lithic`
+- `nash`
+- `pleo`
+- `replicate`
+- `resend`
+- `safebase`
+- `sardine`
+- `shopify`
+- `stripe`
+- `stych`
+- `svix` (that's us!)
+- `zoom`
+
+Additionally there's the option to configure a `Source` as `unverified` to skip
+performing signature verification. This is useful for providers that have no
+verification scheme and for providers whose verification scheme is not yet
+supported by Ingest.
+
+:::tip
+Let us know if your webhook provider isn't yet supported so we can add it!
+:::
+
+
+## Create a Source
+
+Getting started with Ingest is a matter of configuring a `Source`.
+
+```sh
+curl https://api.svix.com/ingest/api/v1/source \
+  -H "Authorization: Bearer $SVIX_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d @- << EOF
+{
+  "name": "from-github",
+  "description": "receives webhooks from GitHub",
+  "uid": "from-github",
+  "type": "github",
+  "config": {
+    "secret": "<the signing secret>"
+  }
+}
+EOF
+```
```

Comment:
This is not doing any syntax highlighting 🤔 

<img width="759" alt="Screenshot 2024-12-04 at 5 04 22 PM" src="https://github.com/user-attachments/assets/9062f15c-ea75-4518-b6ff-ac355e5cd426">

Reply:
changed from `sh` to `shell` and I see some colors now.